usersync/chooser_test.go

```diff
@@ -463,25 +529,23 @@ func TestChooserEvaluate(t *testing.T) {
 			givenSyncTypeFilter:         syncTypeFilter,
 			normalizedBidderNamesLookup: openrtb_ext.NormalizeBidderName,
 			expectedSyncer:              nil,
-			expectedEvaluation:          BidderEvaluation{Bidder: "AppNexus", SyncerKey: "keyA", Status: StatusBlockedByGDPR},
+			expectedEvaluation:          BidderEvaluation{Bidder: "AppNexus", SyncerKey: "keyA", Status: StatusBlockedByPrivacy},
 		},
 		{
-			description:                 "Case Insensitivity Check For Blocked By CCPA",
```

Comment:
Maybe we should keep this case insensitivitiy test case?

Reply:
I removed it primarily because we removed the CCPA check from the `evaluate()` logic, thus making this test case unnecessary.